src/traffic_server/traffic_server.cc

```diff
@@ -1753,6 +1767,12 @@ main(int /* argc ATS_UNUSED */, const char **argv)
   command_index = find_cmd_index(command_string);
   command_valid = command_flag && command_index >= 0;
 
+  // Attach point when TS is blocked for debugging is in this loop.
```

Comment:
Would it make sense doing this only if `DEBUG`?  I mean just to make it extremely obvious.

Reply:
In some cases the test case failure will only happen in optimized code, so I'd prefer to have this available without --enable-debug .